src/include/grpc_transcoding/response_to_json_translator.h

```diff
@@ -58,22 +58,29 @@ namespace transcoding {
 //       an incomplete message at the end of the input. The callers will need to
 //       detect it and act appropriately.
 //
+
+struct JsonResponseTranslateOptions {
+  ::google::protobuf::util::JsonPrintOptions json_print_options;
```

Comment:
Add comment for each option variable,  here is the place users will look for their usage.
You can move the comment from line 74 to here.